db/Task.go

```diff
@@ -22,12 +22,13 @@ type Task struct {
 	Diff   bool `db:"diff" json:"diff"`
 
 	// override variables
-	Playbook    string  `db:"playbook" json:"playbook"`
-	Environment string  `db:"environment" json:"environment"`
-	Limit       string  `db:"hosts_limit" json:"limit"`
-	Secret      string  `db:"-" json:"secret"`
-	Arguments   *string `db:"arguments" json:"arguments"`
-	GitBranch   *string `db:"git_branch" json:"git_branch"`
+	Playbook         string   `db:"playbook" json:"playbook"`
+	Environment      string   `db:"environment" json:"environment"`
+	Limit            *string  `db:"hosts_limit" json:"limit"`
+	Secret           string   `db:"-" json:"secret"`
+	Arguments        *string  `db:"arguments" json:"arguments"`
+	RemovedArguments []string `db:"-" json:"removed_arguments"`
```

Comment:
How this related to `--limit`?

Reply:
I can not merge this PR, I don't understand what this PR do. This changes can break current usage. We have flag `Task.AllowOverrideArgsInTask` but it is ignored in the PR.

You need `--limit` right? Lets create separate PR for `--limit` and I will merge it after review.